### DISCUSSION TITLE
Stoplight generated documentation is not alphabetically sorted

### DISCUSSION THREAD

**Discussion by CreepPork** [dedoc/scramble#80] [Ideas]
Hi!

At the moment, I have a bunch of APIs and it's in a random order, making it quite difficult trying to read the docs.

I think that this might be both a Stoplight issue and a Scramble issue, because I think if the OpenAPI schema was sorted alphabetically, then it would show it sorted in Stoplight.

**Comment by romalytvynenko** [comment]
@CreepPork sorry for the late reply!

Will make it happen in 0.8.0.

Here is the issue for it, you can track it if needed: https://github.com/dedoc/scramble/issues/144

🙌
